webapp/plugins/vue-filters.js

```diff
@@ -81,7 +81,8 @@ export default ({ app = {} }) => {
 
       return contentExcerpt
     },
-    proxyApiUrl: url => {
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T16:25:38Z" title="Tuesday, February 18th 2020, 5:25:38 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -82,6 +82,7 @@ export default ({ app = {} }) => {
       return contentExcerpt
     },
     proxyApiUrl: url => {
+      url = url && (url.url || url)
```


very unclear to me what's going on here... could we call the input... `input`? and set it as `url` when there is no `input.url`? or am I reading this wrong?